src/modules/app/constants/helpers.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
could we rename this to something more declarative? for example removeLinkMarking..., because it is only one helper